### ISSUE TITLE
Define the Skill levels in the UserID Profile

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#736]

**Comment by edickens** [comment]
The UserID has a lot of skill levels, but we need examples so that SCs know how to allocate them to UserIDS.  Also, we need to know how and where they will be used.

**Comment by AlOneill** [comment]
I agree that we need to explain the Skill levels: I have no idea what my skill level should be from just looking at the list -- but perhaps my SC does!!
